build/ProjectReunion-CI.yml

```diff
@@ -138,6 +137,9 @@ jobs:
   condition: in(dependencies.BuildMRT.result, 'Succeeded')
   pool:
     vmImage: 'windows-latest'
+  variables:
+    minimumExpectedTestsExecutedCount: 500  # Sanity check for minimum expected tests to be reported
```

Comment:
500? We have that many that routinely run? (or more actually, 500 is the "minimum expected")

Reply:
This variable is not used it seems.  It is in the commented out code below so i'll just remove it.